examples/api-provider-sample/README.md

```diff
@@ -27,7 +27,7 @@ The source code is laid out in the `src/` tree as follows:
 
 - `gotd.d.ts` — the TypeScript definition of the `gotd` API object that plugins import to interact with the "Greeting of the Day" service
 - `plugin/` — the API initialization script and the implementation of the API objects (`GreetingExt` and similar interfaces).
-  All code in this directory runs exclusively in the separate plugin-host Node process, isolated from the main Theia process.
+  All code in this directory runs exclusively in the separate plugin-host Node process, isolated from the main Theia process, together with either headless plugins or the backend of VS Code plugins.
```

Comment:
`in the separate plugin-host Node process`

I wonder if the use of `the` will mislead the reader into thinking there is only one plugin-host Node process. Using `a` may equally mislead the reader in thinking that the code will run in a Node process dedicated to that code. Of course, neither is true.

- there is a single plugin-host process for all headless plugins running in an app
- there is a plugin-host process for each frontend that attaches to the app's backend. Each one houses all traditional VS Code plugins, servicing their respective frontend only.

How to succinctly express that here...oof. I leave that up to you. 😅